modules/plugins/utility/multicursors/multicursors.nix

```diff
@@ -0,0 +1,126 @@
+{lib, ...}: let
+  inherit (lib.types) bool int str;
+  inherit (lib.nvim.types) mkPluginSetupOption;
+  inherit (lib.options) mkOption mkEnableOption;
+  hintConfig = {lib, ...}: {
+    options = {
+      float_opts = mkOption {
+        description = "The options for the floating hint window";
+        type = lib.types.submodule {
+          options = {
+            border = mkOption {
+              type = lib.types.str;
+              default = "none";
+              description = "The border style for the hint window";
+            };
+          };
+        };
+      };
+      position = mkOption {
+        type = lib.types.str;
+        default = "bottom";
+        description = "The position of the hint window";
```

Comment:
You're already inheriting some types from `lib` at the top. You can move those type definitions to the inherit, and refer to them by name directly.